Apps/WebClient/src/ClientApp/src/store/modules/patientData/utils.ts

```diff
@@ -37,6 +38,22 @@ export function getPatientDataRecordState(
         }
     );
 }
+export function isAllPatientDataTypesStored(
```

Comment:
Nit: the function should be named "areAllPatientDataTypesStored" for grammatical consistency.